### PR TITLE
refactor(bpapi): Decorate remote procedure calls

### PR DESCRIPTION
**Pull request by k32**

### CODE REVIEW

File: apps/emqx_prometheus/src/proto/emqx_prometheus_proto_v1.erl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        ]).
+
+-include_lib("emqx/include/bpapi.hrl").
+
+introduced_in() ->
+    "5.0.0".
+
+-spec start([node()]) -> emqx_rpc:multicall_result().
+start(Nodes) ->
+    rpc:multicall(Nodes, emqx_prometheus, do_start, []).
```
Comment (qzhuyan) [inline apps/emqx_prometheus/src/proto/emqx_prometheus_proto_v1.erl:34]: maybe with a timeout.

File: apps/emqx_dashboard/src/emqx_dashboard_collection.erl [hunk 2]
```diff
@@ -52,6 +52,11 @@ start_link() ->
 
 get_collect() -> gen_server:call(whereis(?MODULE), get_collect).
 
+-spec select_data() -> [#mqtt_collect{}].
+select_data() ->
+    Time = emqx_dashboard_collection:get_universal_epoch() - 7200000,
```
Comment (qzhuyan) [inline apps/emqx_dashboard/src/emqx_dashboard_collection.erl:57]: Where the 7200000 comes from?

Comment (k32) [reply apps/emqx_dashboard/src/emqx_dashboard_collection.erl:57]: This constant comes from the old code, I just moved this function here

File: apps/emqx_prometheus/src/proto/emqx_prometheus_proto_v1.erl (hunk 1 above)
Comment (k32) [reply apps/emqx_prometheus/src/proto/emqx_prometheus_proto_v1.erl]: Fixed.

**Review by qzhuyan — APPROVED** [review]

**Pull request merged**